build.gradle

```diff
@@ -174,7 +175,7 @@ project(':iceberg-bundled-guava') {
 
   shadowJar {
     classifier null
-    configurations = [project.configurations.compileOnly]
+    configurations = [project.configurations.compileClasspath]
```

Comment:
Is this correct? We include `compileOnly` to avoid including `compile` or `implementation` dependencies that are expected in the runtime classpath.

Reply:
yes that's correct. I made sure that the generated Jars are identical

Reply:
this is because Compile is now basically compileOnly

![image](https://user-images.githubusercontent.com/413025/128029813-a99f4cb6-aefb-48dd-a826-869e22364143.png)